packages/opencensus-propagation-b3/tsconfig.full.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Do both these tsconfig files need to exist?

Reply:
I think it was an outdated version.